src/history-provider.ts

```
@@ -35,6 +35,16 @@ import {
   buildBufferScalarSubquery,
   buildBufferObjectSubquery,
 } from './utils/buffer-sql-builder';
+import {
+  stageBufferTable,
+  BufferStagingSource,
+} from './utils/buffer-staging';
```

ESLint

[error] Replace `⏎··stageBufferTable,⏎··BufferStagingSource,⏎` with `·stageBufferTable,·BufferStagingSource·`

(prettier/prettier)